app/views/pages/change_ECF_training_provider.html.erb

```diff
@@ -0,0 +1,112 @@
+<% content_for :title, "Change ECF-based training provider for ECTs and their mentors" %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+
+    <h1 class="govuk-heading-l">Change ECF-based training provider for ECTs and their mentors</h1>
+
+<ul class="govuk-list">
+  <li>
+    [Change training provider for ECTs and mentors across 1 or 2 years](#change-training-provider-for-ects-and-mentors-across-1-or-2-years)
```

Comment:
Sadly we can't use markdown here, so these need to be converted to `<a href="" class="govuk-link"> ... </a>`.